circle.yml

```diff
@@ -1,14 +1,16 @@
 machine:
   xcode:
-    version: "7.2"
+    version: "9.2.0"
 
 dependencies:
   pre:
     - brew install ios-sim
+    - ios-sim showdevicetypes
```

Comment:
Why do we need `ios-sim showdeivetypes`?

Reply:
want to confirm the list of supported device in circle ci. Since the list is different with local ones.

Reply:
I see.
Can you remove this?